utils/sinv.f.in

```diff
@@ -0,0 +1,106 @@
+!-----------------------------------------------------------------------
```

Comment:
filename should be sinv.f90.in, not sinv.f.in

Reply:
Okay, just following guidance from above.